pydatastructs/graphs/algorithms.py

```diff
@@ -548,3 +550,68 @@ def strongly_connected_components(graph, algorithm):
         "isn't implemented for finding strongly connected components."
         %(algorithm, graph._impl))
     return getattr(algorithms, func)(graph)
+
+
+def random_walk(graph, start, path_length, alpha = None):
+    """
+    Executes random walk from the starting node and given graph.
+
+    Parameters
+    ==========
+
+    graph: Graph
+        The graph on which random walk needs to implemented.
+    start: Starting GraphNode
+        The node/vertex from where random walk will begin.
+    path_length:
+        The path length of random walk.
+    alpha:
+        A parameter which determines the randomness. More the value, more random the path will 
+        be and less chances of restart.
+
+    Returns
+    =======
+
+    components: list
```

Comment:
```suggestion
    path: list
```
`components` are for `minimum_spanning_tree`.